packaging/suse/make_spec.sh

```diff
@@ -101,13 +101,16 @@ for file in manifests/*.yaml; do
   # fix image name
   sed -e "s|image:[ ]*sles12/\(.*\):|image: %{_base_image}/\1:|g" -i %{buildroot}/%{_datadir}/%{name}/\$file
 done
+# Install registry-configuration file
+install -d %{buildroot}/%{_datadir}/%{name}/config/registry
+install -D -m 0644 config/registry/registry-config.yaml %{buildroot}/%{_datadir}/%{name}/config/registry/registry-config.yaml
```

Comment:
we should probably add the new folder/file as well in the `%files` section

Reply:
I think the file is covered by `%{_datadir}/%{name}/*` already.